### ISSUE TITLE
Add stub for vcomp100.dll._vcomp_fork

### ISSUE THREAD

**Issue by DarkPlayer** [wine-compholio/wine-staging#26]
RFC for "Add stub for vcomp100.dll._vcomp_fork"

|  |  | 
| ------------- | ------------- |
| **Bug**        | https://bugs.winehq.org/show_bug.cgi?id=31640 |
| **Patch**      | http://kegel.com/wine/openmp/ |

AJ does not accept stub implementations for this function although they fix a lot of games / applications. The stubs will cause the OpenMP section to be run in a single thread instead of multiple threads.

**Comment by slackner** [comment]
I have just rebased the rebased and cleaned up the patches. Added a missing DECL_HIDDEN and also fixed one of the specfile entries.

-> http://ix.io/g0W

Even though its just a stub so far, I think we should add it (just waiting for testbot results).

As soon as we have the threadpool worker implementation in place, we might want to change it / add patches on top of it, but at the moment its definitely better than nothing. ;)
